modules/nodes-custom-scheduler-packaging.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-custom-scheduler.adoc
+
+[id="nodes-custom-scheduler-packaging_{context}"]
+= Packaging and deploying a custom scheduler
+
+To include a custom scheduler in your cluster, create an image and include that image in a deployment.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created a scheduler binary.
++
+[NOTE]
+====
+Information on how to create the scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation.
+====
+
+.Procedure
+
+. Create a container image containing the scheduler binary.
+
+.. Create a Dockerfile:
++
+----
+FROM <source_image>
+ADD <path_to_binary> /usr/local/bin/kube-scheduler
+----
+
+.. From the directory that contains the Dockerfile, build the image:
++
+----
+$ docker build -t <dest_env_registry_ip>:<port>/<namespace>/<image_name>:<tag>
+----
+
+.. From the directory that contains the Dockerfile, push the image to a registry:
++
+----
+$ docker push <dest_env_registry_ip>:<port>/<namespace>/<image_name>:<tag>
+----
+
+. Deploy the custom scheduler.
+
+.. Create a file that contains the deployment resources for the custom scheduler:
++
+.Example `custom-scheduler.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: custom-scheduler
+  namespace: kube-system <1>
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: custom-scheduler
+subjects:
+- kind: ServiceAccount
+  name: custom-scheduler
+  namespace: kube-system <1>
+roleRef:
+  kind: ClusterRole
+  name: system:kube-scheduler
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: custom-scheduler
+  namespace: kube-system <1>
+  labels:
+    app: custom-scheduler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: custom-scheduler
+  template:
+    metadata:
+      labels:
+        app: custom-scheduler
+    spec:
+      serviceAccount: custom-scheduler
+      containers:
+        - name: custom-scheduler
+          image: "<namespace>/<image_name>:<tag>" <2>
+          imagePullPolicy: Always
+----
+<1> This procedure uses the `kube-system` namespace, but you can use the namespace of your choosing.
+<2> Specify the container image that you created for the custom scheduler.
+
+.. Create the deployment resources in the cluster:
```

Comment:
Before this step there are couple more steps which needs to be performed. Below are the details

we need the content from "If RBAC is enabled on your cluster" under "Enable leader Election" here https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/ with out which we cannot get any pod using custom-scheduler.